frontend/components/how-it-work/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { GradientText } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const HowItWork = ({ title, paragraph, cardDetails, sideImage }) => {
+  const test = new Array(6);
+  test.fill(1);
```

Comment:
this should an array from backend you dont have to make the content type but the compoenet must take the array as a a prop